smd_min_entry.js

```diff
@@ -28,9 +28,12 @@ export {
 	// Attr enum
 	HREF,
 	SRC,
-	LANG,
+	CLASS,
 	CHECKED,
 	START,
+	// Renderer
+	default_renderer,
+	logger_renderer,
```

Comment:
The whole point of the min entry is to not include the example renderers.
They exist for starting point during development by "having something on the screen" before you go and make your own.
Normal entry you use with a bundler that will tree-shake them.
The min entry is usable from a cdn and should only have whats necessary.

Reply:
Alright, not exposing the `logger_renderer` then.